### ISSUE TITLE
Compound Individual Page

### ISSUE THREAD

**Issue by parinaznasr** [bhklab/PharmacoDB-JS#385]
Highest priority: Annotated target
- [x] Annotated Targets - wrong table is used. (Show the list of targets:  symbol + ensbmelId , it doesn't need to have datasets,)
- [x]  Synonyms table: Rephrase "PharmacoGx" to something like "Standardized name in PharmacoSet"
- [x] Cell Line Summary table: Click on the number of experiments to take the user to cell line vs compound page for the given compound and cell line.
- [x] Tissue Summary table: Click on the number of experiments to take the user to tissue vs compound page for the given compound and tissue.
- [x] AAC(Cell lines) - add slider and show zoomed in text 
- [x] Tissue Summary table - add a pie chart 
- [x] Molecular Feature table: replace cryptic feature types (Kallisto...)
- [x] Molecular Features table: Use gene symbol for the gene column
- [x] Molecular Feature table: sorted by stat by default
- [x] Molecular Feature table: rename "stat" column to "Sensitivity Metric" 
- [x] Drop 'Significant by Permutation Test' column
- [x] Molecular Feature table: color the rows based on the correlation value (for example: red for positive, blue for negative) 
- [x] Show msg for NA sect

**Comment by sisiranair** [comment]
- [ ] The conclusion was not to remove PharmacoGx from Synonyms, but to rephrase it to something like "Standardized name in PharmacoSet"

**Comment by sisiranair** [comment]
- [ ] AAC(Cell lines) - add slider and show zoomed in text 
- [ ] Tissue Summary table - add a pie chart
